tests/templates/test_integration.py

```diff
@@ -176,558 +138,347 @@
+               {'weights': qml.init.strong_ent_layers_uniform(n_layers=3, n_wires=2), 'wires': range(2)}),
+              (qml.templates.StronglyEntanglingLayers,
+               {'weights': qml.init.strong_ent_layers_uniform(n_layers=2, n_wires=3), 'wires': range(3)}),
+              (qml.templates.StronglyEntanglingLayers,
+               {'weights': qml.init.strong_ent_layers_normal(n_layers=3, n_wires=2), 'wires': range(2)}),
+              (qml.templates.StronglyEntanglingLayers,
+               {'weights': qml.init.strong_ent_layers_normal(n_layers=2, n_wires=3), 'wires': range(3)}),
+              (qml.templates.RandomLayers,
+               {'weights': qml.init.random_layers_uniform(n_layers=3, n_rots=2, n_wires=1), 'wires': range(1)}),
+              (qml.templates.RandomLayers,
+               {'weights': qml.init.random_layers_uniform(n_layers=3, n_rots=2, n_wires=2), 'wires': range(2)}),
+              (qml.templates.RandomLayers,
+               {'weights': qml.init.random_layers_normal(n_layers=2, n_rots=2, n_wires=1), 'wires': range(1)}),
+              (qml.templates.RandomLayers,
+               {'weights': qml.init.random_layers_normal(n_layers=2, n_rots=2, n_wires=2), 'wires': range(2)}),
+              (qml.templates.QAOAEmbedding,
+               {'features': [1.], 'weights': qml.init.qaoa_embedding_uniform(n_layers=3, n_wires=1),
+                'wires': range(1)}),
+              (qml.templates.QAOAEmbedding,
+               {'features': [1., 2.], 'weights': qml.init.qaoa_embedding_uniform(n_layers=3, n_wires=2),
+                'wires': range(2)}),
+              (qml.templates.QAOAEmbedding,
+               {'features': [1.], 'weights': qml.init.qaoa_embedding_normal(n_layers=2, n_wires=1),
+                'wires': range(1)}),
+              (qml.templates.QAOAEmbedding,
+               {'features': [1., 2.], 'weights': qml.init.qaoa_embedding_normal(n_layers=2, n_wires=2),
+                'wires': range(2)}),
+              (qml.templates.QAOAEmbedding,
+               {'features': [1.], 'weights': qml.init.qaoa_embedding_normal(n_layers=2, n_wires=1),
+                'wires': range(1)}),
+              (qml.templates.QAOAEmbedding,
+               {'features': [1., 2.], 'weights': qml.init.qaoa_embedding_uniform(n_layers=2, n_wires=2),
+                'wires': range(2)}),
+              (qml.templates.SimplifiedTwoDesign,
+               {'initial_layer_weights': qml.init.simplified_two_design_initial_layer_uniform(n_wires=4),
+                'weights': qml.init.simplified_two_design_weights_uniform(n_layers=3, n_wires=4),
+                'wires': range(4)}),
+              (qml.templates.SimplifiedTwoDesign,
+               {'initial_layer_weights': qml.init.simplified_two_design_initial_layer_normal(n_wires=4),
+                'weights': qml.init.simplified_two_design_weights_normal(n_layers=3, n_wires=4),
+                'wires': range(4)}),
+              (qml.templates.BasicEntanglerLayers,
```

Comment:
The FIXTURE dictionary could include any (non-diff or diff) arguments. But since I want to test the init functions, I take all keyword arguments with their default values. The ``rot`` argument should be tested in the template's unit test class elsewhere...